airflow/models/base.py

```diff
@@ -38,3 +38,11 @@
 # used for typing
 class Operator:
     pass
+
+
+def get_id_collation_args():
+    collation = conf.get('core', 'sql_engine_collation_for_ids', fallback=None)
+    if collation:
+        return {'collation': collation}
+    else:
+        return {}
```

Comment:
```suggestion
   return {'collation': collation} if collation else {}
```

Reply:
Also I'm wondering if response of this function will change over time... Maybe it would be easier to use:
```python
COLLATION_ARGS = get_id_collation_args()
```
and import this value everywhere?

Reply:
Agree